source/Table/Table.js

```diff
@@ -315,7 +320,7 @@ export default class Table extends PureComponent {
           autoContainerWidth
           className={cn('ReactVirtualized__Table__Grid', gridClassName)}
           cellRenderer={this._createRow}
-          columnWidth={width}
+          columnWidth={totalColumnWidth}
```

Comment:
This should prob be `Math.max(width, totalColumnWidth)` no? Since `Table` columns are supposed to grow to fit.